pkgs/applications/editors/atom/default.nix

```
@@ -54,7 +54,7 @@ let
 
     preFixup = ''
       gappsWrapperArgs+=(
-        --prefix "PATH" : "${gvfs}/bin"
+        --prefix "PATH" : "${glib}/bin"
```

```suggestion
        --prefix "PATH" : "${glib.bin}/bin"
```

just to be certain. and can a comment also be added? (for gio)

---

Something like this?

---

yep